PowerShell/BuildingBlocks/CustomerAndResources/UpdateCustomProperty.ps1

```diff
@@ -0,0 +1,15 @@
+﻿# Initialize the session
+.\Init.ps1
+
+# Retrieve a customer and create an extended property associated to the customer
+$customer = Get-BT_Customer -Ticket $mspc.Ticket -CompanyName "Default"
+$extendedProperty = Add-BT_ExtendedProperty -Ticket $mspc.WorkgroupTicket -Name "ExampleProperty" -Value "Custom Value" -ReferenceEntityType "Customer" -ReferenceEntityId $customer.Id
```

Comment:
In the latest copy (i sync today on 10/16), Add can only be called once. Second call generated error 

```
Add-BT_ExtendedProperty : Create failed with '{"$type":"System.Collections.Generic.Dictionary`2[[System.String,
mscorlib],[System.String, mscorlib]], mscorlib","WorkgroupOrganizationId":"feb0ac98-93d3-4d5c-9606-7e412b586ab2","Name"
:"ExampleProperty","Type":"String","ReferenceEntityId":"0a14a417-970f-11e6-9467-00155d002056"}'
At line:1 char:21
+ ... dProperty = Add-BT_ExtendedProperty -Ticket $mspc.WorkgroupTicket -Na ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (BitTitan.Powers...xtendedProperty:AddBTExtendedProperty) [Add-BT_Exte
   ndedProperty], InvalidOperationException
    + FullyQualifiedErrorId : AddBTExtendedProperty,BitTitan.Powershell.Cmdlets.AddBTExtendedProperty
```

Reply:
Yep, it can only be called once. 

You're right that having an output of "Old value..." can cause some confusion as to whether you can use the Add-BT_ExtendedProperty cmdlet to also update existing custom properties. I'll change the output to just say "Value..." and the update value to say "Updated value...".